src/controller_examples/simple_controller/spec/reconciler.rs

```diff
@@ -82,7 +82,6 @@ pub open spec fn make_config_map(cr: CustomResourceView) -> ConfigMapView
     let config_map = ConfigMapView::default()
         .set_metadata(ObjectMetaView::default()
             .set_name(cr.metadata.name.get_Some_0() + new_strlit("-cm")@)
-            .set_namespace(cr.metadata.namespace.get_Some_0())
```

Comment:
The `namespace` is not set because you don't want to compare it with the request's namespace in `handle_create_req`?

Reply:
Yes.
Also, it is unnecessary to set namespace here: if you don't set it, k8s API will set it; if you set it wrongly, the request will be rejected.

Reply:
> if you don't set it, k8s API will set it

When will k8s API set it?

Reply:
Do you mean in real execution? After the k8s API server receives the creation request, either the k8s API server or etcd will fill the namespace if that is empty. I think it is likely etcd (https://pkg.go.dev/go.etcd.io/etcd/clientv3/namespace) but I am not entirely sure. But I am sure that after the creat request gets handled, if you get the object you will see the namespace set in the object.

Reply:
> But I am sure that after the create request gets handled, if you get the object you will see the namespace set in the object.

I only want to confirm this.